config/data-api.yml

```diff
@@ -156,6 +165,54 @@ components:
           type: string
           description: A human-readable description of the export
           example: Daily export of applications for TAD
+        updated_at:
+          type: string
+          format: date-time
+          description: Time of last change
+          example: 2021-05-20T12:34:00Z
+    ParameterInvalidResponse:
```

Comment:
Great to have these. Maybe we should look at standardising these across APIs, @cwrw?

Reply:
Agree, we are currently refactoring/tidying up validations and errors on the vendor api, when that work is done we def should try and review what we can share/standardise